packages/botonic-plugin-flow-builder/src/content-fields/whatsapp-button-list/flow-whatsapp-button-list-row.tsx

```diff
@@ -0,0 +1,26 @@
+import { ContentFieldsBase } from '../content-fields-base'
+import { HtWhatsappButtonListRow } from '../hubtype-fields'
+
+export class FlowWhatsappButtonListRow extends ContentFieldsBase {
+  public title = ''
+  public description = ''
+  public target?: string
+
+  static fromHubtypeCMS(
+    component: HtWhatsappButtonListRow,
+    locale: string
+  ): FlowWhatsappButtonListRow {
+    const newRow = new FlowWhatsappButtonListRow(component.id)
+    newRow.title = ContentFieldsBase.getTextByLocale(locale, component.text)
+    newRow.description = ContentFieldsBase.getTextByLocale(
+      locale,
+      component.description
+    )
+    newRow.target = component.target?.id
+    return newRow
+  }
+
+  toBotonic() {
```

Comment:
Add return type (the `Row` that it's already declared in `flow-whatsapp-button-list-section.tsx`)